diff --git a/test/e2e_source/helpers/kafka_helper.go b/test/e2e_source/helpers/kafka_helper.go
index 70435f92f..3cb550eee 100644
--- a/test/e2e_source/helpers/kafka_helper.go
+++ b/test/e2e_source/helpers/kafka_helper.go
@@ -45,7 +45,8 @@ const (
 	strimziUserResource  = "kafkausers"
 	interval             = 3 * time.Second
 	timeout              = 4 * time.Minute
-	kcatImage            = "quay.io/openshift-knative/kcat:1.7.1"
+	#kcatImage            = "quay.io/openshift-knative/kcat:1.7.1"
+	kafkaCatImage        = "image-registry.openshift-image-registry.svc:5000/local-images/kcat:1.7.1"
 )
 
 var (
@@ -96,7 +97,7 @@ func MustPublishKafkaMessage(client *testlib.Client, bootstrapServer string, top
 	}
 	args = append(args, "-l", "/etc/mounted/payload")
 
-	client.T.Logf("Running kcat %s", strings.Join(args, " "))
+	client.T.Logf("Running kafkacat %s", strings.Join(args, " "))
 
 	pod := corev1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
@@ -105,9 +106,9 @@ func MustPublishKafkaMessage(client *testlib.Client, bootstrapServer string, top
 		},
 		Spec: corev1.PodSpec{
 			Containers: []corev1.Container{{
-				Image:   kcatImage,
+				Image:   kafkaCatImage,
 				Name:    cgName + "-producer-container",
-				Command: []string{"kcat"},
+				Command: []string{"kafkacat"},
 				Args:    args,
 				VolumeMounts: []corev1.VolumeMount{{
 					Name:      "event-payload",
